core/src/main/java/org/apache/accumulo/core/data/Key.java

```diff
@@ -107,6 +107,17 @@ public Key(Text row) {
   }
 
   /**
+   * Creates a key with the specified row, empty column family, empty column qualifier, empty column visibility, timestamp {@link Long#MAX_VALUE}, and delete
+   * marker false.
```

Comment:
I'll do it. I wonder if this is the right behaviour or if the user would expect the byte array to be used as it is. In my version of this that I'm using at work, I'm setting `copy` to false all the time because I know that's safe considering how I'm using the arrays. When I submitted this patch, I decided to switch to true because it was more safe. Maybe we should add this parameter to each constructor using `byte[]` explicitly so that the user knows that he can avoid copying the array?